store/tikv/region_cache.go

```diff
@@ -285,9 +286,9 @@ func (c *RPCContext) String() string {
 		c.Region.GetID(), c.Meta, c.Peer, c.Addr, c.PeerIdx)
 }
 
-// GetTiKVRPCContext returns RPCContext for a region. If it returns nil, the region
+// GetRPCContext returns RPCContext for a region. If it returns nil, the region
 // must be out of date and already dropped from cache.
-func (c *RegionCache) GetTiKVRPCContext(bo *Backoffer, id RegionVerID, replicaRead kv.ReplicaReadType, followerStoreSeed uint32) (*RPCContext, error) {
+func (c *RegionCache) GetRPCContext(engine kv.StoreType, bo *Backoffer, id RegionVerID, replicaRead kv.ReplicaReadType, followerStoreSeed uint32) (*RPCContext, error) {
```

Comment:
Seems tiflash is not handled by this function. So I still prefer `GetTiKVRPCContext`.

Reply:
tiflash logic will set `engine == kv.TiFlash` and use this logic and it's the PR's purpose

Reply:
But there is still `GetTiFlashRPCContext`.

Reply:
oh, `GetTiFlashRPCContext` need be remove